ouroboros-network/test/Test/Ouroboros/Network/Diffusion/Node/NodeKernel.hs

```diff
@@ -63,6 +63,9 @@ import qualified Ouroboros.Network.Testing.ConcreteBlock as ConcreteBlock
 import           Simulation.Network.Snocket (AddressType (..),
                      GlobalAddressScheme (..))
 
+import           Test.QuickCheck (Arbitrary(..), oneof, chooseInt, choose)
+import Data.IP (IP(..), toIPv4, toIPv6)
```

Comment:
Could you run `scripts/ci/check-stylish-network.sh` locally (it's added back to ci in #3558).